src/docs/releases/1.6.0.md

```diff
@@ -4,6 +4,28 @@ Release date: Not yet released
 
 ## Breaking Changes
 
+### Users
+
+The `OrchardCore.Users` module now provides multiple new permissions for added control on user related functions. 
+
+In previous versions, `View Users` permission was enough to show the `Users` menu item. However, with the newly added permissions, `ListUsers` or at least one `ListUsersInRole_{0}` variation is required to see the `Users` menu. The `View Users` permission now only allows you to view user's profile. Here is a list of the new permissions
+
```

Comment:
And then update the doc if some related updates are done.

E.g. as commented if we only keep the existing `ViewUsers` and not define the new `ListUsers`.

And have the variation `ViewUsersInRole_{0}`.

Reply:
updated.